coil-core/src/androidInstrumentedTest/kotlin/coil3/decode/AndroidDecoderTest.kt

```diff
@@ -235,79 +268,75 @@ class BitmapFactoryDecoderTest {
     }
 
     @Test
-    fun exifOrientationPolicy_ignore() = runTest(timeout = 1.minutes) {
-        val factory = BitmapFactoryDecoder.Factory(
-            exifOrientationPolicy = ExifOrientationPolicy.IGNORE,
-        )
+    fun exifOrientationPolicy_ignore() = runTestPerDecoder(timeout = 1.minutes, ExifOrientationPolicy.IGNORE) {
+        // Android ImageDecoder handle exif internally so we cannot tune it
+        // Test BitmapFactoryDecoder only
+        assumeTrue(factory() is BitmapFactoryDecoder.Factory)
```

Comment:
I think this test should still be able to run for `ImageDecoder`. Each of the 8 images in the test have differently rotation + flipped values. If `ImageDecoder` handles exif rotation internally then its output should match the expected non-rotated image for each case.

Reply:
I mean, ImageDecoder always behaves like ExifOrientationPolicy.RESPECT_PERFORMANCE
if user prefer other ExifOrientationPolicy, we should fallback to BitmapFactoryDecoder as well

Reply:
Ah I see - sorry I though it always behaved like `ExifOrientationPolicy.IGNORE`. I'm not a huge fan that we allow this to be configured so this might be a push to remove this API in Coil 3 if `ImageDecoder` can't support it.